common/scripted_triggers/wc_portrait_triggers.txt

```diff
@@ -244,6 +245,10 @@ has_yaungol_portrait_trigger = {
 	exists = culture
 	is_race_no_gene_trigger = { RACE = creature_yaungol }
 }
+has_yaungol_portrait_trigger = {
+	exists = culture
+	is_race_no_gene_trigger = { RACE = creature_lich }
+}
```

Comment:
Duplicate.

Reply:
This was an oversight, I renamed it to `has_lich_portrait_trigger` in b39fff224466497201cf1050022d51a5fa8a5e4d.